gnosis/eth/clients/blockscout_client.py

```diff
@@ -40,7 +40,7 @@ class BlockscoutClient:
         EthereumNetwork.REI_CHAIN_TESTNET: "https://scan-test.rei.network/",
         EthereumNetwork.METER_MAINNET: "https://scan.meter.io/",
         EthereumNetwork.METER_TESTNET: "https://scan-warringstakes.meter.io/",
-        EthereumNetwork.GODWOKEN_TESTNET_V1: "https://v1.betanet.gwscan.com/",
+        EthereumNetwork.GODWOKEN_TESTNET_V1: "https://v1.testnet.gwscan.com/",
```

Comment:
Is this blockscout api compatible?

Reply:
I'm not sure and I don't have time to test it unfortunately. I just updated old one which is probably deprecated to new one.

Reply:
Please make sure, if not remove it